src/views/product-downloads-view/components/release-information/index.tsx

```diff
@@ -28,35 +28,40 @@ const NoteCard = ({ selectedRelease }) => {
 	const currentProduct = useCurrentProduct()
 	const { name, shasums, shasums_signature, version } = selectedRelease
 	return (
-		<InlineAlert
-			className={s.alert}
-			color="neutral"
-			title="Note"
-			description={
-				<>
-					You can find the{' '}
-					<InlineLink
-						href={`https://releases.hashicorp.com/${name}/${version}/${shasums}`}
-						textSize={200}
-					>
-						SHA256 checksums for {currentProduct.name} {version}
-					</InlineLink>{' '}
-					online and you can{' '}
-					<InlineLink
-						href={`https://releases.hashicorp.com/${name}/${version}/${shasums_signature}`}
-						textSize={200}
-					>
-						verify the checksums signature file
-					</InlineLink>{' '}
-					which has been signed using{' '}
-					<InlineLink href="https://www.hashicorp.com/security" textSize={200}>
-						{"HashiCorp's GPG key"}
-					</InlineLink>
-					.
-				</>
-			}
-			icon={<IconInfo16 className={s.cardIcon} />}
-		/>
+		<>
+			<InlineAlert
```

Comment:
Since InlineAlert is the only element we don't need to wrap it in a fragment.

Reply:
Missed this in the clean up!